workflow/schemas/config.schema.yaml

```diff
@@ -8,7 +8,10 @@ definitions:
   filterentry:
     type: object
     additionalProperties:
-      type: [string, "null"]
+      anyOf:
+        - type: string
+        - type: "null"
+        - type: object
```

Comment:
The object type here should be constrained to contain extra and and expression (both string).